src/net/peacefulcraft/cowbot/handlers/ModChatMessageHandler.java

```diff
@@ -13,7 +13,7 @@ public static boolean isReplayMessage(String username, String message) {
 
   public static void handle(Message message) {
     if (!message.getAuthor().isPresent()) { return; }
-    String author = message.getAuthor().get().getUsername();;
+    String author = message.getAuthor().get().getUsername();
```

Comment:
also removed the double semicolon here because I noticed it and couldn't help myself